src/main/scala/org/neo4j/spark/util/Validations.scala

```diff
@@ -110,6 +110,8 @@ object Validations {
             s"You need to set the ${Neo4jOptions.RELATIONSHIP_TARGET_LABELS} option")
         }
         case QueryType.QUERY => {
+          ValidationUtil.isFalse(neo4jOptions.query.value.matches("(?s).*(LIMIT \\d+|SKIP ?\\d+)\\s*\\z"),
```

Comment:
i guess we need to do the matches as insensitive?

Reply:
please also add a test with the skip/limit as snake case and lowercase

Reply:
Note sure this is fixed.  It complains wherever I put "LIMIT" in this query and query.count does not limit results:

`relationship_cypher="""
MATCH (s)-[r:IS_SIMILAR]-(t)
RETURN s.name AS source
,t.name AS target
,r.similarity AS similarity
,s.communityId AS louvain_community_id
"""
relsDf2=spark.read.format("org.neo4j.spark.DataSource") \
    .option("query", relationship_cypher) \
    .option("url", neo4j_url) \
    .option("authentication.type", "basic") \
    .option("authentication.basic.username", neo4j_user) \
    .option("authentication.basic.password", neo4j_password) \
    .option("query.count", 2) \
    .load()
    
relsDf2`